autoscheduler/scraper/views.py

```diff
@@ -40,14 +42,14 @@ class RetrieveTermView(generics.ListAPIView):
         This view returns all the terms
     """

```

Comment:
The changes you made broke the sort function somehow at least for me, here's what my term list looks like now:
![image](https://user-images.githubusercontent.com/28655462/99750070-57722180-2aa5-11eb-815b-2d44e8b663b8.png)
Do you have the same problem? It seems to order terms for a semester correctly for around half of them which is why the test with terms for 2020 didn't break

Reply:
Yep the same is happening to me, will look into it